### ISSUE TITLE
New blog entry

### ISSUE THREAD

**Issue by k0rn1** [Oberon00/Oberon00.github.io#1]
I would like to know if the blog will be continued in the future

**Comment by Oberon00** [comment]
Yes, I do have some ideas for posts I'd like to write, so the blog is not death, only frozen. ATM I'm just too busy with other things.
